crates/cheatcodes/src/utils.rs

```diff
@@ -111,6 +112,27 @@ impl Cheatcode for getLabelCall {
     }
 }
 
+impl Cheatcode for computeCreateAddressCall {
+    fn apply(&self, _state: &mut Cheatcodes) -> Result {
+        let Self { nonce, deployer } = self;
+        Ok(deployer.create(nonce.to()).abi_encode())
```

Comment:
`.to()` panics if the number does not fit the target type (u64), can you add a check before this? You can copy it from `chainIdCall`

Reply:
sure